awx/conf/views.py

```diff
@@ -135,7 +139,11 @@ def perform_destroy(self, instance):
             setting.delete()
             settings_change_list.append(setting.key)
         if settings_change_list:
-            connection.on_commit(lambda: handle_setting_changes.delay(settings_change_list))
+            connection.on_commit(lambda: clear_setting_cache.delay(settings_change_list))
+            if any([setting.startswith('LOG_AGGREGATOR') for setting in settings_change_list]):
+                # call notify to rsyslog. no data is need so payload is empty
+                with pg_bus_conn() as conn:
+                    conn.notify('rsyslog_configurer', "")
```

Comment:
I wasn't sure if this was missed or left intentionally - should this just call the new function too?

Reply:
It was missed. Thought I had already changed this over, fixing it now.